client/client.go

```diff
@@ -0,0 +1,1375 @@
+	Client OpenFgaClient
+
+	options *ClientGetStoreOptions
+}
+
+type ClientGetStoreOptions struct {
+}
+
+func (request SdkClientGetStoreRequest) Options(options ClientGetStoreOptions) SdkClientGetStoreRequest {
+	request.options = &options
+	return request
+}
+
+func (request SdkClientGetStoreRequest) Execute() (openfga.GetStoreResponse, *_nethttp.Response, error) {
+	return request.Client.GetStoreExecute(request)
+}
+
+func (client *OpenFgaClient) GetStoreExecute(request SdkClientGetStoreRequest) (openfga.GetStoreResponse, *_nethttp.Response, error) {
+	return client.OpenFgaApi.GetStore(request.ctx).Execute()
+}
+
+func (client *OpenFgaClient) GetStore(ctx _context.Context) SdkClientGetStoreRequest {
+	return SdkClientGetStoreRequest{
+		Client: *client,
+		ctx:    ctx,
+	}
+}
+
+// / DeleteStore
+type SdkClientDeleteStoreRequest struct {
+	ctx    _context.Context
+	Client OpenFgaClient
+
+	body    *ClientDeleteStoreRequest
+	options *ClientDeleteStoreOptions
+}
+
+type ClientDeleteStoreRequest struct {
+	User             string            `json:"user,omitempty"`
+	Relation         string            `json:"relation,omitempty"`
+	Object           string            `json:"object,omitempty"`
+	ContextualTuples *[]ClientTupleKey `json:"contextual_tuples,omitempty"`
+}
+
+type ClientDeleteStoreOptions struct {
+	AuthorizationModelId *string `json:"authorization_model_id,omitempty"`
+}
+
+type ClientDeleteStoreResponse struct{}
+
+func (request SdkClientDeleteStoreRequest) Options(options ClientDeleteStoreOptions) SdkClientDeleteStoreRequest {
+	request.options = &options
+	return request
+}
+
+func (request SdkClientDeleteStoreRequest) Body(body ClientDeleteStoreRequest) SdkClientDeleteStoreRequest {
+	request.body = &body
+	return request
+}
+
+func (request SdkClientDeleteStoreRequest) Execute() (ClientDeleteStoreResponse, *_nethttp.Response, error) {
+	return request.Client.DeleteStoreExecute(request)
+}
+
+func (client *OpenFgaClient) DeleteStoreExecute(request SdkClientDeleteStoreRequest) (ClientDeleteStoreResponse, *_nethttp.Response, error) {
+	httpResponse, err := client.OpenFgaApi.DeleteStore(request.ctx).Execute()
+	return ClientDeleteStoreResponse{}, httpResponse, err
+}
+
+func (client *OpenFgaClient) DeleteStore(ctx _context.Context) SdkClientDeleteStoreRequest {
+	return SdkClientDeleteStoreRequest{
+		Client: *client,
+		ctx:    ctx,
+	}
+}
+
+/* Authorization Models */
+
+// / ReadAuthorizationModels
+type SdkClientReadAuthorizationModelsRequest struct {
+	ctx    _context.Context
+	Client OpenFgaClient
+
+	body    *ClientReadAuthorizationModelsRequest
+	options *ClientReadAuthorizationModelsOptions
+}
+
+type ClientReadAuthorizationModelsRequest struct {
+	User             string            `json:"user,omitempty"`
+	Relation         string            `json:"relation,omitempty"`
+	Object           string            `json:"object,omitempty"`
+	ContextualTuples *[]ClientTupleKey `json:"contextual_tuples,omitempty"`
+}
```

Comment:
ReadAuthorizationModelsRequest does not need any of these configuration options. Just the storeID is enough